Docker-Swarm-deployment/single-node/lip/tests/requirements.txt

```diff
@@ -1,4 +1,4 @@
 pika==1.2.0
-redis==3.5.3
+redis==4.5.3
```

Comment:
for this version, need to update following code of lines in latest-ingestion-pipeline-test.py

import redis
r = redis.Redis(host='localhost',port=6381, decode_responses=True, password='password')
x = r.json().get('iisc_ac_in_89a36273d77dac4cf38114fca1bbe64392547f86_rs_iudx_io_surat_itms_realtime_information_surat_itms_live_eta')